<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipleItemsContextMenu</class>
 <widget class="QMenu" name="MultipleItemsContextMenu">
  <addaction name="InfoAction"/>
  <addaction name="DeleteAction"/>
  <addaction name="CropAction"/>
  <addaction name="GroupAction"/>
  <addaction name="separator"/>
  <addaction name="DelDupsAction"/>
  <addaction name="DelUnavailableAction"/>
  <addaction name="separator"/>
  <addaction name="SelectionSubmenu"/>
  <addaction name="separator"/>
  <addaction name="ExportAction"/>
  <addaction name="ConvertAction"/>
  <addaction name="separator"/>
  <addaction name="CopyToClipboardAction"/>
  <addaction name="ShowStatisticAction"/>
  <action name="InfoAction">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/info.png</normalon>
    </iconset>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
  </action>
  <action name="DeleteAction">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/delete.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="CropAction">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/crop.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="GroupAction">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/group.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Group together</string>
   </property>
  </action>
  <action name="DelDupsAction">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/delduplicates.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Remove duplicates in</string>
   </property>
  </action>
  <action name="DelUnavailableAction">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/delete.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Delete unavailable in</string>
   </property>
  </action>
  <widget class="QMenu" name="SelectionSubmenu">
   <property name="title">
    <string>Select</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/playlist/select.png</normalon>
    </iconset>
   </property>
   <addaction name="SelRipOffsAction"/>
   <addaction name="SelSameTypesAction"/>
   <addaction name="SelSameFilesAction"/>
   <addaction name="SelFoundAction"/>
  </widget>
  <action name="SelRipOffsAction">
   <property name="text">
    <string>Rip-offs in this items</string>
   </property>
  </action>
  <action name="SelSameTypesAction">
   <property name="text">
    <string>Items with the same types</string>
   </property>
  </action>
  <action name="SelSameFilesAction">
   <property name="text">
    <string>Items from the same files</string>
   </property>
  </action>
  <action name="SelFoundAction">
   <property name="text">
    <string>Found in this items</string>
   </property>
  </action>
  <action name="CopyToClipboardAction">
   <property name="text">
    <string>Copy all paths</string>
   </property>
  </action>
  <action name="ShowStatisticAction">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/statistic.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show statistic</string>
   </property>
  </action>
  <action name="ExportAction">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/export.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Export selected</string>
   </property>
  </action>
  <action name="ConvertAction">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/convert.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Convert selected</string>
   </property>
  </action>
 </widget>
</ui>
